### PR TITLE
feat!: implement backend and fspiop event services

### PR DESCRIPTION
**Pull request by kirgene**
New API handlers:
* POST /bulkTransactions:
  produces:
    * SDKOutboundBulkRequestReceived
* PUT /bulkTransactions/{bulkTransactionId}:
  produces:
   * SDKOutboundBulkAcceptPartyInfoReceived
   * SDKOutboundBulkAcceptQuoteReceived

New event handlers:
* PartyInfoRequested: (GET /party, PUT /party)
* BulkQuotesRequested: (POST /bulkQuotes, PUT /bulkQuotes)
* SDKOutboundBulkAcceptPartyInfoRequested
* SDKOutboundBulkAcceptQuoteRequested

### CODE REVIEW

**Review by kleyow — COMMENTED** [review]
Most of it looks good to me atm. Just address the CI issues.

File: modules/api-svc/src/config.js [hunk 1]
```diff
@@ -92,27 +92,27 @@ module.exports = {
     backendEventHandler: {
         enabled: env.get('ENABLE_BACKEND_EVENT_HANDLER').default('true').asBool(),
         domainEventConsumer: {
-            brokerList: env.get('BACKEND_EVENT_CONSUMER_BROKER_LIST').default('localhost:9092').asString(),
+            brokerList: env.get('BACKEND_EVENT_CONSUMER_BROKER_LIST').default('kafka:9092').asString(),
```
Comment (mdebarros) [inline modules/api-svc/src/config.js:95]: I would prefer these to default to `local` and rather have them set to `kafka` as part of integration tests, etc.

Same comment applies elsewhere.

**Review by mdebarros — COMMENTED** [review]
LGTM, just a minor comment for you to consider.

**Review by mdebarros — APPROVED** [review]
+1

**Pull request merged**